webroot/src/store/license/license.spec.ts

```diff
@@ -140,17 +156,27 @@ describe('License Store Mutations', () => {
     });
 });
 describe('License Store Actions', async () => {
-    it('should successfully start licensees request', async () => {
+    it('should successfully start licensees request with next page', async () => {
         const commit = sinon.spy();
-        // const getters = sinon.spy();
         const dispatch = sinon.spy();
         const params = { getNextPage: true };
 
         await actions.getLicenseesRequest({ commit, getters, dispatch }, { params });
 
         expect(commit.calledOnce).to.equal(true);
         expect(commit.firstCall.args).to.matchPattern([MutationTypes.GET_LICENSEES_REQUEST]);
-        expect(dispatch.calledThrice).to.equal(true);
+        expect(dispatch.callCount).to.equal(4);
+    });
+    it('should successfully start licensees request with previous page', async () => {
+        const commit = sinon.spy();
+        const dispatch = sinon.spy();
+        const params = { getPrevPage: true };
+
+        await actions.getLicenseesRequest({ commit, getters, dispatch }, { params });
+
+        expect(commit.calledOnce).to.equal(true);
+        expect(commit.firstCall.args).to.matchPattern([MutationTypes.GET_LICENSEES_REQUEST]);
+        expect(dispatch.callCount).to.equal(4);
```

Comment:
Is the purpose of having both of these to ensure `getPrevPage` and `getNextPage` do not alter the behavior?

Reply:
Yep